_functions_onlyoffice.sh

```diff
@@ -0,0 +1,109 @@
+  delete_docker_volume ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME}
+  echo_info "Done."
+  echo_info "Onlyoffice data dropped"
+}
+
+do_create_onlyoffice() {
+  if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
+    echo_info "Creation of the OnlyOffice Docker volume ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} ..."
+    create_docker_volume ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME}
+    echo_info "OnlyOffice Docker volume ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} created"
+  fi
+}
+
+do_stop_onlyoffice() {
+  echo_info "Stopping OnlyOffice ..."
+
+  ensure_docker_container_stopped ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME}
+  echo_info "OnlyOffice container ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} stopped."
+}
+
+do_start_onlyoffice() {
+  echo_info "Starting OnlyOffice..." 
+  echo_info "Starting OnlyOffice container ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} based on image ${DEPLOYMENT_ONLYOFFICE_IMAGE}:${DEPLOYMENT_ONLYOFFICE_IMAGE_VERSION}"
+
+  # Ensure there is no container with the same name
+  delete_docker_container ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME}
+
+  ${DOCKER_CMD} run \
+    -d \
+    -p "127.0.0.1:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}:80" \
+    -v ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME}:/var/log/onlyoffice  \
+    --name ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} ${DEPLOYMENT_ONLYOFFICE_IMAGE}:${DEPLOYMENT_ONLYOFFICE_IMAGE_VERSION}
+
+  echo_info "${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} container started"
+
+  check_ONLYOFFICE_availability
+}
+
+check_ONLYOFFICE_availability() {
+  echo_info "Waiting for Onlyoffice DocumentServer availability on port ${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
+  local count=0
+  local try=600
+  local wait_time=1
+  local RET=-1
+
+  local temp_file="/tmp/${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME}_${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}.txt"
+
+  while [ $count -lt $try -a $RET -ne 0 ]; do
+    count=$(( $count + 1 ))
+    set +e
+
+    curl -s -q --max-time ${wait_time} http://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT} > ${temp_file}
+    RET=$?
+    if [ $RET -ne 0 ]; then
+      [ $(( ${count} % 10 )) -eq 0 ] && echo_info "OnlyOffice documentserver not yet available (${count} / ${try})..."
+    else
+      curl -f -s --max-time ${wait_time} http://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}/healthcheck > ${temp_file} 
+      local status=$(grep "true" ${temp_file})
+      if [ "${status}" == "true" ]; then
+        RET=0   
+      fi
+    fi
+
+    if [ $RET -ne 0 ]; then
+      echo -n "."
+      sleep $wait_time
+    fi
+    set -e
+  done
+  if [ $count -eq $try ]; then
+    echo_error "Onlyoffice DocumentServer ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} not available after $(( ${count} * ${wait_time}))s"
+    exit 1
+  fi
+  echo_info "Onlyoffice DocumentServer ${DEPLOYMENT_ONLYOFFICE_CONTAINER_NAME} up and available"
+}
+
+# #############################################################################
+# Env var to not load it several times
+_FUNCTIONS_DATABASE_LOADED=true
+echo_debug "_functions_database.sh Loaded"
```

Comment:
```suggestion
echo_debug "_functions_onlyoffice.sh Loaded"
```

Reply:
Ok will be updated.
_function_es.sh will require equivalent change 
https://github.com/exoplatform/adt/blob/master/_functions_es.sh#L138-L139

Reply:
Updated with right value